main.cpp

```diff
@@ -52,84 +52,84 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:    cat
-//  action 1:
+//  Noun:     cat
+//  action 1: the cat purrs
 cat.purr();
-//  action 2:
+//  action 2: the cat runs
 cat.run();
-//  action 3:
+//  action 3: the cat sleeps
 cat.sleep();
 //  2)
-//  Noun:    dog
-//  action 1:
+//  Noun:     dog
+//  action 1: the dog barks
 dog.bark();
-//  action 2:
+//  action 2: the dog growls
 dog.growl();
-//  action 3:
+//  action 3: the dog bites
 dog.bite();
 //  3)
-//  Noun:    car
-//  action 1:
+//  Noun:     car
+//  action 1: the car goes
 car.go();
-//  action 2:
+//  action 2: the car stops
 car.stop();
-//  action 3:
+//  action 3: the car leaks fluid
 car.leakFluid();
 //  4)
-//  Noun:    leg
-//  action 1:
+//  Noun:     leg
+//  action 1: the leg kicks
 leg.kick();
-//  action 2:
+//  action 2: the leg flexs
 leg.flex();
-//  action 3:
+//  action 3: the leg bends
 leg.bend();
 //  5)
-//  Noun:    mouth
-//  action 1:
+//  Noun:     mouth
+//  action 1: the mouth speaks
 mouth.speak();
-//  action 2:
+//  action 2: the mouth tastes
 mouth.taste();
-//  action 3:
+//  action 3: the mouth blows
 mouth.blow();
 //  6)
-//  Noun:    finger
-//  action 1:
+//  Noun:     finger
+//  action 1: the finger points
 finger.point();
-//  action 2:
+//  action 2: the finger scratches
 finger.scratch();
-//  action 3:
+//  action 3: the finger rubs
 finger.rub();
 //  7)
-//  Noun:    wife
-//  action 1:
+//  Noun:     wife
+//  action 1: the wife cooks
 wife.cook();
-//  action 2:
+//  action 2: the wife cleans
 wife.clean();
-//  action 3:
+//  action 3: the wife sleeps
 wife.sleep();
 //  8)
-//  Noun:    friend
-//  action 1:
+//  Noun:     friend
+//  action 1: the friend gives a hi-five
 friend.giveHiFive();
-//  action 2:
+//  action 2: the friend fights
 friend.fight();
-//  action 3:
+//  action 3: the friend plays
 friend.play();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.